packages/graphql/src/schemaTypes/objectTypes/gql-Query.ts

```diff
@@ -79,7 +79,7 @@ export const Query = objectType({
 
     t.nonNull.boolean('isGlobalMode', {
       description: 'Whether the app is in global mode or not',
```

Comment:
> Global mode should be determined by whether a project is specified or not. The --global flag can be used to force global mode, but if there is not a project, we can assume the Launchpad is being run in global mode.

@chrisbreiding Gah I was missing this context for global mode when I did [#22073](https://github.com/cypress-io/cypress/pull/22073/files#diff-a29a2817a2313d8351e2c932721e30fc122da6f8189a558bb7beadd15443172c). Would you mind adding your explanation to the description here so it doesn't get lost in the future? 🙏